tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/wwwroot/app/matter/createMatter.controller.js

```diff
@@ -1974,6 +2025,13 @@
 
                 }
 
+                var additionalFields = getAdditionalMatterPropertiesFieldsData();

```

Comment:
Get extra matter fields values and pass to controller on create and notify button click as part of third step while  assign content type to matter  and at same time we assign new content type to matter with additional field properties columns values.